src/library-authoring/create-library/data/apiHooks.test.tsx

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { renderHook, waitFor } from '@testing-library/react';
+
+import { mockContentLibrary } from '@src/library-authoring/data/api.mocks';
+import { initializeMocks } from '@src/testUtils';
+import { libraryAuthoringQueryKeys } from '../../data/apiHooks';
+import {
+  useCreateLibraryRestore,
+  useCreateLibraryV2,
+  useGetLibraryRestoreStatus,
+} from './apiHooks';
+import { LibraryRestoreStatus } from './restoreConstants';
+
+mockContentLibrary.applyMock();
+const { axiosMock } = initializeMocks();
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+const wrapper = ({ children }: { children: React.ReactNode }) => (
+  <QueryClientProvider client={queryClient}>
+    {children}
+  </QueryClientProvider>
+);
+
+describe('create library apiHooks', () => {
+  beforeEach(() => {
+    queryClient.clear();
+  });
+
+  afterEach(() => {
+    axiosMock.reset();
+  });
```

Comment:
```suggestion
  beforeEach(() => {
    queryClient.clear();
    axiosMock.reset();
  });
```
Similar in practice but more concise.